ci/build.yml

```diff
@@ -197,6 +197,8 @@ core unix regular build:
   stage: build
   <<: *gitlab_caching
   needs: []
+  variables:
```

Comment:
@onvej-sl I am curious - will the thp switch be available in "frozen" build too, when it is defined here in core unix regular?